src/inmanta/server/agentmanager.py

```diff
@@ -636,23 +637,48 @@ async def trigger_agent(self, env: UUID, agent_id: str) -> Apireturn:
         raise NotImplementedError()
 
     @protocol.handle(methods.list_agent_processes)
-    async def list_agent_processes(self, environment: Optional[UUID], expired: bool) -> Apireturn:
+    async def list_agent_processes(
+        self,
+        environment: Optional[UUID],
+        expired: bool,
+        start: Optional[UUID] = None,
+        end: Optional[UUID] = None,
+        limit: Optional[int] = None,
+    ) -> Apireturn:
+        """List all agent processes whose sid is after start and before end
+
+        :param environment: Optional, the environment the agent should come from.
+        :param expired: If True, expired agents will also be shown, they are hidden otherwise.
+        :param start: The sid all of the selected agent process should be greater than this, defaults to None
+        :param end: The sid all of the selected agent process should be smaller than this, defaults to None
+        :param limit: Whether to limit the number of returned entries, defaults to None
+        :raises BadRequest: Limit, start and end can not be set together
```

Comment:
This comment should be removed

Reply:
Did it in https://github.com/inmanta/inmanta/pull/2521/commits/541aebaff4c0a488a53e0082c7783e45da04f8d6